modules/gdscript/gdscript.cpp

```diff
@@ -285,7 +285,11 @@ void GDScript::_get_script_method_list(List<MethodInfo> *r_list, bool p_include_
 #endif
 				mi.arguments.push_back(arginfo);
 			}
-
+#ifdef TOOLS_ENABLED
+			for (int i = 0; i < func->get_default_arg_values().size(); i++) {
+				mi.default_arguments.push_back(func->get_default_arg_values()[i]);
+			}
```

Comment:
Doesn't this work?
```suggestion
			mi.default_arguments.append_array(func->get_default_arg_values());
```

Reply:
Thanks for the reminder, this is indeed the more concise way